nuitka/Options.py

```diff
@@ -2067,51 +2067,40 @@ def isMacOSUiElementApp():
 
 def _getPythonFlags():
     """*list*, values of ``--python-flag``"""
-    # There is many flags, pylint: disable=too-many-branches
-
-    # singleton, pylint: disable=global-statement
     global _python_flags
 
     if _python_flags is None:
         _python_flags = set()
 
+        flag_dict = {
+            "no_site": ["-S", "nosite", "no_site"],
```

Comment:
Also, why do people use lists, when tuples will do, wastes memory and speed I think, does it not?